packages/containers/src/DeleteResource/sagas.js

```diff
@@ -75,6 +75,7 @@ export function* deleteResourceValidate(
 			yield put({
 				type: deleteResourceConst.DIALOG_BOX_DELETE_RESOURCE_SUCCESS,
 				model: {
+					...get(action, 'data.model', {}),
```

Comment:
Don't you want to put the model in some attribute like 'model' in order to not have override of the id / labelResource if the entity has those fields ?
```suggestion
					model: get(action, 'data.model', {}),
```

Reply:
If you do that, we have a model inception: model.model.{}. It's weird

Reply:
I prefer spread all model properties in model in model and override id, labelResource to avoid a breaking change